index.html

```diff
@@ -1,12 +1,40 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>To review</title>
-</head>
-<body>
-    
-</body>
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <h1 id="title">TITLE</h1>
+    <p id="description">This is our project for microverse ....</p>
```

Comment:
Please wrap your `<h1>` and  `<p>` in a `head` tag

Reply:
@anderCM 
- [ ] Good job :+1:  for using [GitHub inline comments feature](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) for suggested changes, But please make sure you have included an explanation, a "why," for each requested change. 
- For example:
***Why don't you use an `header` for the `h1` and `p` instead of using `div`, that would improve the HTML structure and make the page more accessible.***

Reply:
Thanks for feedback, I am adding explanation and more details about my comments